content/platforms/kubernetes/concepts/persistent-volumes.md

```diff
@@ -37,8 +37,12 @@ requirements]({{< relref "/rs/administering/designing-production/hardware-requir
 To explicitly specify the persistent storage size, use the *volumeSize*
 property as described in the example above.
 
+{{< warning >}}
+Be aware the persistent volume size cannot be changed after deployment. Trying to change this value after deployment can result in breaking the operator. Please be sure your specified *volumeSize* is correct at the time of creation.
```

Comment:
```suggestion
Be aware the persistent volume size cannot be changed after deployment. Trying to change this value after deployment can result in unexpected. Please be sure your specified *volumeSize* is correct at the time of creation.
```
Unexpected or undefined behavior rather than "break the operator".